backend/app/routes/faq/index.js

```diff
@@ -0,0 +1,26 @@
+const {authMiddleware, generateJWT} = require('../../../helpers/middlewares/auth');
+const express = require ("express");
+const FAQ = require('../../models/faq');
+const router = express.Router();
+
+
+router.use('/', authMiddleware , async (req,res,next) => {
```

Comment:
You need to add the Validation schema, check other routes for it
Also extract the function to a seperate file

Reply:
This is pending